modules/fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/fs2/Main.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.fs2
+
+import cats.effect.{ExitCode, IOApp, IO}
+
+object Main extends IOApp {
+  def run(args: List[String]): IO[ExitCode] =
+    EnrichJobConfig.command.parse(args) match {
+      case Right(cfg) =>
+        for {
+          environment <- Environment.init[IO](cfg).value
+          exit <- environment match {
+            case Right(e) => e.use { env =>
+              val enrich = Enrich.run[IO](env)
+              val updates = AssetsRefresh.run[IO](env)
+              enrich.merge(updates).compile.drain.as(ExitCode.Success)
```

Comment:
Will this `.as(ExitCode.Success)` ever be used?

Reply:
Only if our `Stream` ends... which will never happen, in other words - no.